funccount.py

```diff
@@ -0,0 +1,30 @@
+import os
+import signal
+import subprocess as sp
+import time
+
+class FUNCCOUNT:
+    def __init__(self, funcs, sleep=0):
+        self.funcs = funcs
+        self.sleep = sleep
+
+    def start(self):
+        cmd = f"sleep {self.sleep} && funccount {self.funcs}"
+
+        self.run = sp.Popen(cmd, shell=True, stdout=sp.PIPE, text=True,
+                            start_new_session=True)
+
+    def stop(self):
+        pgid = os.getpgid(self.run.pid)
+        os.killpg(pgid, signal.SIGINT)
```

Comment:
Do we need this nasty process group stuff? Possibly if it's not in a shell we can just use self.proc.send_signal()...

self.run seems like a poor choice for a variable name

Reply:
I couldn't get it to work without the process group stuff. I also tried doing this outside of Python but sending a `kill -INT <pid>` didn't work. strace shows this:

```
0x7ffdebf1be20, 0, NULL)      = ? ERESTARTSYS (To be restarted if SA_RESTART is set)
--- SIGINT {si_signo=SIGINT, si_code=SI_USER, si_pid=93677, si_uid=0} ---
rt_sigreturn({mask=[CHLD]})             = -1 EINTR (Interrupted system call)
```